### ISSUE TITLE
Meta Page Footer

### ISSUE THREAD

**Issue by ghost** [Dental-Revenue/Codebase-2-Dev#820]
**Describe the bug**
The meta pages (accessibility, privacy policy, sitemap, terms of use) are missing the footer.
